modules/windows/README.md

```diff
@@ -326,3 +336,7 @@ give you clues as to why the collector isn't working.
   ```bash
   ./go.d.plugin -d -m windows
   ```
+
+### Slow response
+
+When windows exporter is slow to answer queries from `go.d.plugin`, it is possible the collector will stop the module. To avoid this set a higher `timeout` value for the `job`.
```

Comment:
Doesn't look helpful here. Also, I'd say it is not the correct recommendation because (due to ND specifics) it is better to increase both `update_every` and `timeout` (assuming timeout deadline exceeded is not a one-time problem (e.g. on start/first scrape)).

Reply:
I decided to remove the new section.